### PR TITLE
profile json added

### PR DESCRIPTION
**Pull request by theMillenniumFalcon**

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

File: public/data/theMillenniumFalcon.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "http://github.com/theMillenniumFalcon",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/nishankstwt",
+      "icon": "twitter"
+    }
+  ]
+}
```
Comment (nhcarrigan) [inline public/data/theMillenniumFalcon.json:17]:
```suggestion
}

```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Just a quick nitpick for EOF NL

File: public/data/theMillenniumFalcon.json (hunk 1 above)
Comment (theMillenniumFalcon) [reply public/data/theMillenniumFalcon.json]: new-line added

**Comment by nhcarrigan** [comment]
This will need a rebase:

https://www.nhcarrigan.com/notes/#/git-commands/rebase-branch

**Comment by eddiejaoude** [comment]
Closing as seems like another PR was created https://github.com/EddieHubCommunity/LinkFree/pull/271

**Pull request closed without merging**